ossdbs/fem/mesh.py

```diff
@@ -31,8 +34,14 @@ def __init__(self, geometry: netgen.occ.OCCGeometry, order: int) -> None:
 
     def generate_mesh(self, meshing_parameters: dict) -> None:
         """Generate NGSolve mesh."""
-        netgen_mp = self._meshing_parameters(meshing_parameters)
+        netgen_mp = self._meshing_parameters(meshing_parameters["MeshingHypothesis"])
         self._mesh = ngsolve.Mesh(self.geometry.GenerateMesh(mp=netgen_mp))
+        if (
+            "HPRefinement" in meshing_parameters
```

Comment:
Is this condition safe? What happens if `HPRefinement` isn't in the dictionary?

Reply:
Yes, if `HPRefinement` is not in the dictionary, simply no hp-refinement is applied